### PR TITLE
Update Scalars tutorial text to be more descriptive.

### PR DESCRIPTION
**Pull request by manivaradarajan**
Linkify some text to Keras docs and clean up text in some minor ways to be more descriptive.

### CODE REVIEW

File: docs/r2/scalars_and_keras.ipynb [hunk 1]
```diff
@@ -207,9 +207,14 @@
         "\n",
         "You're now ready to define, train and evaluate your model. \n",
         "\n",
-        "To log the *loss* scalar as you train, you'll create the Keras TensorBoard callback, specifying a log directory, and pass it to [Model.fit()](https://https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit).\n",
+        "To log the *loss* scalar as you train, you'll do the following:\n",
```
Comment (GalOshri) [inline docs/r2/scalars_and_keras.ipynb]: I don't think "\n" is needed (here and in the following lines). There is also and extra comma.

File: docs/r2/scalars_and_keras.ipynb [hunk 2]
```diff
@@ -207,9 +207,14 @@
...
+        ", specifying a log directory, and pass it \n",
         "\n",
-        "TensorBoard reads log data from the log directory hierarchy. In this notebook, the root log directory is \"logs/scalars\", suffixed by a timestamped subdirectory. This enables easily identification and selection of training runs as you use TensorBoard and iterate on your model.\n",
+        "1.   Create the Keras [TensorBoard callback](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard)\n",
+        "2.   Specify a log directory\n",
+        "3.   Pass the TensorBoard callback to Keras' [Model.fit()](https://https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit).\n",
+        "\n",
+        "TensorBoard reads log data from the log directory hierarchy. In this notebook, the root log directory is ```logs/scalars```, suffixed by a timestamped subdirectory. The timestamped subdirectory enables you to easily identify and select of training runs as you use TensorBoard and iterate on your model.\n",
```
Comment (GalOshri) [inline docs/r2/scalars_and_keras.ipynb]: typo "easily identify and select of training runs"

File: docs/r2/scalars_and_keras.ipynb (hunk 1 above)
Comment (manivaradarajan) [reply docs/r2/scalars_and_keras.ipynb]: I've ensured it looks correct in Colab now.

Comment (manivaradarajan) [reply docs/r2/scalars_and_keras.ipynb]: I've made sure it looks right in Colab now.

File: docs/r2/scalars_and_keras.ipynb [hunk 3]
```diff
@@ -307,15 +311,15 @@
...
+        "You may see TensorBoard display the message \"No dashboards are active for the current data set\". That's because initial logging data hasn't been saved yet. As training progresses, the Keras model will start logging data. TensorBoard will periodicially refresh and show you your scalar metrics. If you're impatient, you can tap the Refresh arrow at the top right.\n",
         "\n",
         "As you watch the training progress, note how both training and validation loss rapidly decrease and then remain stable. In fact, you could have stopped training after 25 epochs, because the training didn't improve much after that point.\n",
         "\n",
         "Hover over the graph to see specific data points. You can also try zooming in with your mouse, or selecting part of them to view more detail.\n",
         "\n",
         "Notice the \"Runs\" selector on the left. A \"run\" represents a set of logs from  a complete round of training, in this case the result of Model.fit(). Developers typically have many, many runs, as they experiment and develop their model over time. \n",
```
Comment (GalOshri) [inline docs/r2/scalars_and_keras.ipynb]: nit: "Complete round of training" - there can be a run from incomplete rounds of training.

Comment (manivaradarajan) [reply docs/r2/scalars_and_keras.ipynb]: Fixed

**Pull request merged**